src/odemis/gui/cont/streams.py

```diff
@@ -122,6 +126,45 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
         self._lbl_exc_peak = None
         self._lbl_em_peak = None
 
+        # Display metadata for integration time, ebeam voltage, probe current and
+        # emission/excitation wavelength
+        if show_md:
+            if isinstance(stream.raw[0], tuple):
+                # Show metadata for first element of raw in case of AR acquisitions or
+                # stitched images
+                md = stream.raw[0][0].metadata
+            else:
+                md = stream.raw[0].metadata
+
+            # Add exposure time and dwell time to integration time
+            if model.MD_EXP_TIME in md.keys() and model.MD_DWELL_TIME in md.keys():
```

Comment:
It should never happen that both EXP_TIME and DWELL_TIME are set simultaneously. If this happens (anyway), just use EXP_TIME. => drop this first if.